packages/node/src/indexer/unfinalizedBlocks.service.spec.ts

```diff
@@ -22,6 +22,7 @@ const makeHeader = (height: number, finalized?: boolean): Header => ({
   blockHeight: height,
   blockHash: hexify(`0xABC${height}${finalized ? 'f' : ''}`),
   parentHash: hexify(`0xABC${height - 1}${finalized ? 'f' : ''}`),
+  timestamp: new Date((1740100000 + height) * 1000),
```

Comment:
Where does `1740100000` come from? Can we move it to a constant with a description